### ISSUE TITLE
Document which components perform network calls

### ISSUE THREAD

**Issue by sureshjoshi** [pantsbuild/scie-pants#39]
For those of us who use [lulu](https://objective-see.org/products/lulu.html) or similar, any new application which calls out to the network needs to be approved.

In existing Pants, it's the system/`pyenv` Python, while now it's `ptex` and standalone Python.

I can tackle this one when I next wipe my rules, so I can give examples.

**Comment by sureshjoshi** [comment]
Appears to only be two items: 

- `ptex` on port 443 located at `{CACHE_DIR}/nce/{digest}/ptex`
- `python3.9` on port 443 located at `{CACHE_DIR}/nce/{digest}...-install_only.tar.gz/python/bin/python3.9`

Afterwards, it's regular Pants stuff

**Comment by jsirois** [comment]
Yeah, that squares. It's ptex to fetch a PBS distribution, then some `pip install`s using that distribution.
